content/en/docs/components/model-registry/getting-started.md

```diff
@@ -132,6 +133,42 @@ ks_client.create(isvc)
 An inference endpoint is now created, using the artifact metadata retrieved from the Model Registry (previous step),
 specifying the serving runtime to be used to serve the model, and references to the original entities in Model Registry.
 
+### Using Model Registry Custom Storage Initializer
+
+The Model Registry Custom Storage Initializer (CSI) is a custom implementation of the KServe Storage Initializer that allows you to use Model Registry metadata to download and deploy models. You can create an InferenceService that references the model and version in the Model Registry:
```

Comment:
```suggestion
The Model Registry Custom Storage Initializer (CSI) is a custom implementation of the KServe Storage Initializer that allows you to use Model Registry metadata to download and deploy models (see [Installation instructions](installation.md)). You can create an InferenceService that references the model and version in the Model Registry:
```

maybe we could add explicit call-out to ensure user has installed this. wdyt?

Reply:
Nice addition, thanks!